templates/partials/scientific-committee-section.html

```diff
@@ -1,158 +1,104 @@
-                        "alt": "University of Toronto",
-                        "icon-class": "fa-solid fa-user"
-                    },
-                    {
-                        "href": "",
-                        "alt": "",
-                        "icon-class": "fab fa-orcid fa-lg"},
-                    {
-                        "href": "",
-                        "alt": "",
-                        "icon-class": "fab fa-researchgate fa-lg"
-                    },
-                    {
-                        "href": "",
-                        "alt": "",
-                        "icon-class": "fab fa-linkedin fa-lg"
-                    },
-                    {
-                        "href": "",
-                        "alt": "GitHub",
-                        "icon-class": "fab fa-github fa-lg"
-                    }
-                ]
+{%  set scientific_commitee_data=[
+    {
+        "name": "Steve Easterbrook",
+        "image": "images/about/scientific/steve-easterbrook.jpg",
+        "expertise": "computer science, machine learning",
+        "organization": {
+            "name": "University of Toronto",
+            "href": "https://www.utoronto.ca/"
+        },
+        "links": []
+    },
+    {
+        "name": "David Huard",
+        "image": "images/about/executive/david-huard.jpg",
+        "expertise": "climate services, climate data analysis",
+        "organization": {
+            "name": "Ouranos",
+            "href": "https://www.ouranos.ca"
+        },
+        "links": [
+            {
+                "href": "https://github.com/huard",
+                "icon_class": "fab fa-github fa-lg"
+            },
+            {
+                "href": "https://www.researchgate.net/profile/David-Huard",
+                "icon_class": "fab fa-researchgate fa-lg"
             },
- {"name": "David Huard",
-    "descriptions":
-        [
             {
-                "title": "Research Specialist",
-                "expertise": ""
+                "href": "https://www.linkedin.com/in/david-huard/",
+                "icon_class": "fab fa-linkedin fa-lg"
             }
-        ],
-      "organization":[
-                {
-                    "name": "Ouranos",
-                    "href": "https://www.ouranos.ca/en",
-                    "icon-class": "fa-solid fa-building-columns"
-                }
-            ],
-    "image": "images/about/executive/david-huard.jpg",
-    "links":
-        [
-            {"text": "Profile",
-            "href": "https://www.ouranos.ca/en/equipe#david-huard-408",
-            "alt": "Ouranos",
-            "icon-class": "fa-solid fa-user"
+        ]
+    },
+    {
+        "name": "H. Damon Matthews",
+        "image": "images/about/scientific/damon-matthews.jpg",
+        "expertise": "climate science, climate modelling, carbon-cycle processes, nature-based climate solutions, remaining carbon budgets, climate targets and scenarios",
+        "organization": {
```

Comment:
Was the expertise overflow in the CSS done to keep the height of the profile cards consistent?

I feel each person's expertise should be shown but there will be a problem with height alignment if some people have more text.